### PR TITLE
Add more functests to init.sls <functests.1node>

### PR DESCRIPTION
**Pull request by jschmid1**
Signed-off-by: Joshua Schmid <jschmid@suse.de>

-----------------

**Checklist:**
- [x] Ran integration tests successfully ( trigger with @susebot run teuthology )

### CODE REVIEW

File: srv/salt/ceph/functests/1node/init.sls [hunk 1]
```diff
@@ -6,3 +6,7 @@ include:
   - .restart.mon
   - .restart.rgw
   - .apparmor
+  - .openstack
+  - .tuned
+  - .remove
+  - .replace
```
Comment (smithfarm) [inline srv/salt/ceph/functests/1node/init.sls]: replace functests already have a separate teuthology test case, so let's drop replace here

**Comment by jschmid1** [comment]
@susebot run teuthology suse/tier1/functional

**Comment by jschmid1** [comment]
@susebot run teuthology suse/tier1/functional

**Comment by jschmid1** [comment]
1pass 1fail(dead)

**Comment by smithfarm** [comment]
@jschmid1 This problem is already solved. The solution is to pre-create the pool before running Stage 3. For this I just need to know the name of the pool, which I'll get by peeking into the source code of the functests you are adding here. Then I'll push a commit that fixes the issue.

**Comment by kshtsk** [comment]
> http://158.69.90.90:8080/job/deepsea-pr/103/
is failed because of OVH failure
@susebot run teuthology with suse/tier1/functional

**Comment by kshtsk** [comment]
@susebot run teuthology with suse/tier1/functional

**Comment by jschmid1** [comment]
@susebot run teuthology with suse/tier1/functional

**Comment by jschmid1** [comment]
> @jschmid1 This problem is already solved. The solution is to pre-create the pool before running Stage 3. For this I just need to know the name of the pool, which I'll get by peeking into the source code of the functests you are adding here. Then I'll push a commit that fixes the issue.

Let's open a separate PR for this

**Pull request merged**